consensus/service/src/consensus_service.rs

```diff
@@ -651,7 +663,7 @@ impl<
                     "sync_status": sync_status,
                     "blocks_behind": blocks_behind,
                     "latest_block_hash": latest_block_hash,
-                    "latest_block_timestamp": latest_block_timestamp,
+                    "latest_block_timestamp": latest_block_timestamp.map_or("".to_string(), |u| u.to_string()),
```

Comment:
`null` is actually more correct, I believe. why change it?

Reply:
Specific request from Blockdaemon